SFDigitalServices/sf-dahlia-web
# Release 06 05 2024 (#2189)

* fix: DAH-2267 Fix the Fix for 'Reset and Start Fresh' option (#2145)

* Angular/DAH-2140 sf cleanup job after test run (#2142)

* DAH-2228 add logs

* DAH-2228 fix logs

* add log

* add another log

* trying something

* DAH-2228 revert

* DAH-2138-prepend-first-names

* DAH-2138 remove unneeded changes

* DAH-2138 update to e2etest-

* DAH-2138 prepend in some names

* DAH-2138 update

* DAH-2138 update 2

* DAH-2138 update 3

* DAH-2140 try adding a step

* DAH-2140 update config

* Updated config.yml

* Updated config.yml

* Updated config.yml

* Updated config.yml

* DAH-2140 clean config

* DAH-2140 add cleanup job

* DAH-2140 add cleanup job to circle config

* DAH-2140 update cleanup job

* DAH-2140 better logs

* DAH-2140 add comments

* DAH-2272 run angular jobs for all non-prod jobs (#2147)

* DAH-2272 run angular jobs for all non-prod jobs

* Updated config.yml

* DAH-2272 run on main

* DAH-2110 scv empty lottery modal new design (#2144)

* DAH-2110 scv empty lottery modal new design

* DAH-2110 add comment

* DAH-2110 phrase

* DAH-2110 fix logic

* DAH-2110 fix plurals

* DAH-2110 update snapshots

* DAH-2110 remove plurals

* DAH-2110 tests

* DAH-2110 componentization, tests, and other updates based on comments

* DAH-2110 remove unneeded file

* fix: DAH-1298 allow searching without leading 0 (#2123)

* DAH-2261 update new design for when a lottery number is entered (#2146)

* DAH-2261 update new design

* DAH-2261 spacing and general public title

* DAH-2261 update snapshot

* DAH-2261 add comments

* DAH-2261 remove unneeded styling

* DAH-2261 componentize the educator listing results row

* DAH-2261 fix lint

* DAH-2261 tests

* DAH-2261 remove next steps link to be handled in another ticket

* DAH-2261 componentize

* feat: DAH-2086: Integrate unleash into Webapp (#2148)

* Feature flag working with userID

* Add useFeatureFlag hook and update HomePage component

* useFeatureFlag enhancements

* Use heroku env variables

* Add in test indicator

* Testing

* streamline useFeatureFlag test

* Test node_env

* use new Unleash env environment variable

* Use Unleash Env variable

* Provide UNLEASH_ENV to jest

* fix: DAH-2219 restore plural SRO info card  (#2137)

* fix: remove extra sro check

* fix: add Room type to sro checks

* fix: lint

* fix: DAH-2275: Remove SFHDC from Housing Counselor Page (#2152)

fix: remove SFHDC

* chore: merging production back into main (#2153)

hotfix: adding email response api and webpage (#2150)

* feature: adding email response api and webpage

* fix: adding .rb extension

* chore: adding logs and test listing

* feat: update yes page content

* chore: following figma styling




* DAH-2078 add metadata for description and title

* DAH-2078 update snapshots

* DAH-2078 remove unneeded log

* DAH-2078 add url

* DAH-2078 add robots

* DAH-2078 add more twitter info

* DAH-2078 pare down meta tags

* DAH-2078 remove unneeded changes

* DAH-2078 update missing imports

* chore: DAH-2296 refactor and test leasing agent support (#2155)

* chore: renaming frontend landing page

* chore: adding tests

* chore: refactoring

* chore: adding comments

* Chore(deps): Bump axios from 0.21.2 to 1.6.8 (#2151)

* fix: DAH-2313 run for all jobs (#2163)

* fix: DAH-2312 do not show base preferences for SCV 1 (#2162)

* DAH-2312 do not show base preferences for SCV 1

* DAH-2312 add listingIsEducatorOne parameter

* feat: DAH-2279 Next Steps Hyperlink (#2156)

* feat: DAH-2314 update lottery video link (#2165)

* feat: update lottery ranking video url

* fix: update snapshots

* DAH-2247 add variable for completed lottery listing (#2160)

* DAH-2247 add variable for completed lottery listing

* DAH-2247 add typing

* DAH-2247 remove cypress downloads

* DAH-2247 update default completed lottery listing id

* DAH-2247 remove unneeded file

* DAH-1420 consistent data only need one listing id (#2171)

* DAH-1420 consistent data only need one listing id

* DAH-1420 turns out you can update lottery numbers

* Chore(deps-dev): Bump rexml from 3.2.5 to 3.2.8 (#2172)

* feat: DAH-2227 Add reset-cache param to Listings API request (#2173)

* feat: DAH-1816 Clean Up React and Veteran Env Vars (#2174)

* feat: DAH-2274 GoogleTranslationService (#2033)

* deploy translate PR env

* feat: testing google cloud auth

* fix: update gemfile -fix-later

* fix: naming

* fix: translate auth

* add function

* feat: DAH-2274 GoogleTranslationService

* fix: DAH-2274 check if result is array

* fix DAH-2274 temp test job

* fix: DAH-2274 add tests

* fix: update gitignore

* fix: remove faraday version change

* fix: update google translate bundle (#2176)

* fix: let faraday version be restricted

* fix: add faraday to gem, test compatibility

* fix: bundle update google-cloud-translate-v2

* feat: DAH-2287: Add Unleash Documentation (#2175)

update readme

* fix: DAH-2227 permit the recache param (#2179)

* Chore(deps): Bump nokogiri from 1.16.2 to 1.16.5 (#2164)

* fix/DAH-2357 fix failing tests (#2184)

* feat: DAH-2337: Set up Account Pages (#2180)

* feat: set up account pages

* fix: remove sign in from shared services

* fix: remove console log

* feat: leasing agent support bayside village (#2185)

* feat: adding bayside village to landing pages

* feat: rails redirect to react

* feat: updating tests

* feat: clean up

* feat: clean up

---------
